examples/src/main/java/com/teamdev/jxbrowser/examples/PrintSettings.java

```diff
@@ -53,6 +53,12 @@ public static void main(String[] args) {
                     .orElseThrow(IllegalStateException::new);
             PrintJob<HtmlSettings> printJob = printer.printJob();
             printJob.settings()
+                    // Configure header with custom text and document title.
+                    .header("<span style=\"font-size: 12px;\">"
```

Comment:
Some HTML uses double quotes, some of it use no quotes at all. Please make it consistent